include/wlr/backend/drm.h

```diff
@@ -33,10 +33,16 @@ bool wlr_output_is_drm(struct wlr_output *output);
  */
 uint32_t wlr_drm_connector_get_id(struct wlr_output *output);
 
+/**
+ * Tries to open non-master fd.
+ * Returns an open file descriptor or -errno on error.
+ */
+int wlr_drm_backend_get_client_fd(struct wlr_backend *backend);
```

Comment:
I wonder if we want this to live in the DRM backend.

We already have a similar function for the DRM dumb allocator, FWIW.

Reply:
I guess we could use the new `backend_get_allocator` functions here